extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -29,41 +29,41 @@ function __construct() {
 	public function updateImageState( $images, $action = '' ) {
 		wfProfileIn( __METHOD__ );
 
-		$deletionList = array();
-		$statsInsert = array();
+		$deletionList = [];
+		$statsInsert = [];
 
-		$sqlWhere = array(
-			ImageReviewStatuses::STATE_APPROVED => array(),
-			ImageReviewStatuses::STATE_REJECTED => array(),
-			ImageReviewStatuses::STATE_QUESTIONABLE => array(),
-		);
+		$sqlWhere = [
+			ImageReviewStatuses::STATE_APPROVED => [],
+			ImageReviewStatuses::STATE_REJECTED => [],
+			ImageReviewStatuses::STATE_QUESTIONABLE => [],
+		];
 
 		foreach ( $images as $image ) {
 			$sqlWhere[ $image['state'] ][] = "( wiki_id = $image[wikiId] AND page_id = $image[pageId]) ";
 
 			if ( $image['state'] == ImageReviewStatuses::STATE_DELETED ) {
-				$deletionList[] = array( $image['wikiId'], $image['pageId'] );
+				$deletionList[] = [ $image['wikiId'], $image['pageId'] ];
 			}
 
-			$statsInsert[] = array(
+			$statsInsert[] = [
 				'wiki_id' => $image['wikiId'],
 				'page_id' => $image['pageId'],
 				'review_state' => $image['state'],
 				'reviewer_id' => $this->user_id
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "reviewer_id" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)